docs/topics/modules/setting_up_debugger.adoc

```diff
@@ -3,12 +3,11 @@
 
 To debug your project code, set up the Che workspace Debugger feature:
 
-. Click the *debug* option from the Debug drop-down.
+. In your workspace, click the *debug* option in the Debug drop-down icon.
```

Comment:
Drop-down icon sounds unusual. I suggest we call these unusual UX elements (Run and Debug) as buttons, as you have done here: https://docs.openshift.io/getting-started-guide.html#running_your_project-hello-world This would ensure consistency as well. 
Ideally, there should have been just an icon with infotips (Run or Debug) on hover, similar to the print icon in docs. Maybe this could go as feedback to the UX folks.